dependencies.yaml

```diff
@@ -493,4 +493,4 @@ dependencies:
       - output_types: [conda, requirements, pyproject]
         packages:
           - dask-cuda==23.4.*
-          - numba>=0.56.2
+          - *numba
```

Comment:
Is this related to the `&numba numba` above?

Reply:
Yes, the `&numba ...` takes a reference to `...` and "stores" it in `numba` which can then be dereferenced and copied in with `*numba`.